src/vignette/storage/static_assets.clj

```diff
@@ -0,0 +1,48 @@
+(ns vignette.storage.static-assets
+  (:require [vignette.storage.protocols :refer :all]
+            [org.httpkit.client :as http]
```

Comment:
Previously I thought your intention was to use the [jet client](https://github.com/mpenet/jet#http-client). I don't know if the interfaces will be compatible between jet <-> httpkit.